src/main/java/bio/terra/cli/command/resources/addref/BqDataset.java

```diff
@@ -18,10 +18,13 @@
 public class BqDataset extends BaseCommand {
   @CommandLine.Mixin ResourceCreation resourceCreationOptions;
 
-  @CommandLine.Option(names = "--project-id", required = true, description = "GCP project id")
+  @CommandLine.Option(names = "--project-id", required = true, description = "GCP project id.")
```

Comment:
maybe indicate that this is the id of the project containing the dataset, not necessarily the project ID of the active workspace.

Reply:
ok